cli/tsc/dts/lib.deno.shared_globals.d.ts

```diff
@@ -348,7 +348,7 @@ declare namespace WebAssembly {
   export function validate(bytes: BufferSource): boolean;
 }
 
-/** Sets a timer which executes a function once after the timer expires. Returns
+/** Sets a timer which executes a function after the delay (in milliseconds). Returns
```

Comment:
Thanks for the contribution! Just would like to clarify this slightly by keeping "once" and adding "elapses".

When describing the JavaScript function "setTimeout," it is more commonly said that the "timer elapses" or the "delay elapses." These phrases convey the idea that the specified time period set by the setTimeout function has passed. This aligns with the typical terminology used in JavaScript documentation and programming discussions.

```suggestion
/** Sets a timer which executes a function once after the delay (in milliseconds) elapses. Returns
```

Reply:
That makes sense, thanks